### PR TITLE
Philosophy 

### PR DESCRIPTION
**Pull request by anovis**
adding section on pcf philosophy

### CODE REVIEW

File: docs/source/sections/philosophy.rst [hunk 1]
```diff
@@ -0,0 +1,32 @@
+=================
+PCF Philosophy
+=================
+
+
+Reduced Infrastructure Operations Burden
+----------------------------------------
+A core principle of PCF is to automate as much of the operations burden as possible. This is done by writing out
+the entire infrastructure as code and tie this code the application. Therefore changes can easily be tested before deploying
```
Comment (davidyum) [inline docs/source/sections/philosophy.rst]: tying* this code to* the application

File: docs/source/sections/philosophy.rst [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+Customizable
+-------------
+There are other IAC options out there, but many times users are limited by what they can do and any customization requires
+digging and editing deep in the source code. We built PCF in python with the idea of easily customizing and overriding any functions
+allowing the user to do complex and unique actions if they would like.
+
+
+Standardize Cloud resources
+---------------------------
+PCF was build with the idea that all cloud resources follow the same basic lifecycles. This makes is much easier to manage
```
Comment (davidyum) [inline docs/source/sections/philosophy.rst]: built*

**Review by davidyum — COMMENTED** [review]
looks good just few typos

File: docs/source/sections/philosophy.rst [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+Customizable
+-------------
+There are other IAC options out there, but many times users are limited by what they can do and any customization requires
+digging and editing deep in the source code. We built PCF in python with the idea of easily customizing and overriding any functions
+allowing the user to do complex and unique actions if they would like.
+
+
+Standardize Cloud resources
+---------------------------
+PCF was built with the idea that all cloud resources follow the same basic lifecycles. This makes is much easier to manage
```
Comment (ethanwlo) [inline docs/source/sections/philosophy.rst]: This makes **it** much easier to manage

**Pull request merged**